#CNV-36463: web console checkups (#76590)
openshift/openshift-docs: modules/virt-latency-checkup-web-console.adoc

// Module included in the following assemblies:
//
// * virt/monitoring/virt-running-cluster-checkups.adoc

:_mod-docs-content-type: PROCEDURE
[id="virt-latency-checkup-web-console_{context}"]
= Running a latency checkup in the web console

Run a latency checkup to verify network connectivity and measure the latency between two virtual machines attached to a secondary network interface.

.Prerequisites

> * You must add a `NetworkAttachmentDefinition`` to the namespace.

Reviewer: 1. Redundant "\`" at the end of `NetworkAttachmentDefinition`.
2. NAD is not enough. The user must also create a bridge interface on the cluster nodes, otherwise the NAD won't connect the VM's secondary interface to any interface on the node, and the job will fail because the VMs won't start.
Please add a reference to https://76590--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-connecting-vm-to-linux-bridge#virt-connecting-vm-to-linux-bridge (it is referenced in the `Additional Resources` section, but at this point of creating a NAD and initiating a checkup, the un-experienced user wouldn't know that this is what they are missing).

Author: Updated.

.Procedure

. Navigate to *Virtualization* -> *Checkups* in the web console.
. Click the *Network latency* tab.
. Click *Install permissions*.
. Click *Run checkup*.

> . Enter a *Name* for the checkup.

Reviewer 2: I don't have access to the UI, but do we need to bold Name here? Or

```suggestion
. Enter a name for the checkup in the *Name* field.
```

> . Select a *NetworkAttachmentDefinition* from the dropdown menu.

Reviewer 2: Per ISG
```suggestion
. Select a *NetworkAttachmentDefinition* from the drop-down menu.
```

. Optional: Set the *Sample duration (seconds)*.
. Optional: Enable *Set maximum desired latency (milliseconds)*, then define the time interval.

> . Optional: Enable *Select nodes* to target specific nodes, then select the *Source node* and *Target node*.

Reviewer 2: Same concern as above. Do we need bold here? Or
```suggestion
. Optional: Set a dureation for (blah) in the *Sample duration (seconds)* field.
. Optional: Enable latency by selecting *Set maximum desired latency (milliseconds)* and defining the time interval.
. Optional: Target specific nodes by selecting *Select nodes* and specifing the *Source node* and *Target node*.
```
I don't have access to the console, so forgive me if this is incorrect.

. Click *Run*.

> You can view the *Status* of the latency checkup in the *Checkups* list on the *Latency checkup* tab. Click on the *Name* of the checkup for more *Latency checkup details*.

Reviewer 2: ```suggestion
You can view the *status of the latency checkup in the *Checkups* list on the *Latency checkup* tab. Click on the *Name* of the checkup for more *Latency checkup details*.
```
